packages/expo-video/src/VideoAirPlayButton.types.ts

```diff
@@ -0,0 +1,39 @@
+import { ViewProps } from 'react-native';
+
+export interface VideoAirPlayButtonProps extends Omit<ViewProps, 'children'> {
+  /**
+   * The color of the button icon while AirPlay sharing is not active.
+   *
+   * @default undefined
+   * @platform ios
+   */
+  tint?: string;
```

Comment:
I think we should provide either an example or a link to what string values are accepted here. There was some internal feedback on how sometimes we lack this information (and I was just discussing today in docs sync to add a guideline in our writing style guide :)). This helps provide clarity to the developer.